spec/system/authentication/redirects_using_referer_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
```

Comment:
This test ensures the redirect after authentication brings the user back to where they started the process. Only applies to login because when registering they will go through the onboarding (already covered by other tests).